src/rust/plugin-sdk/generator-sdk/src/lib.rs

```diff
@@ -0,0 +1,56 @@
+#[cfg(feature = "client")]
+use structopt::StructOpt;
+
+pub mod server;
+
+#[cfg(feature = "client")]
+pub mod client;
+
+#[cfg(feature = "client")]
+/// Configuration for the cache that holds onto plugin client connections
+#[derive(StructOpt, Debug)]
+pub struct ClientCacheConfig {
+    /// The number of concurrent plugin clients to hold
+    /// Defaults to 1000
+    #[structopt(env, default_value = "1000")]
+    pub max_capacity: u64,
+    /// Total amount of time a given entry will live in seconds
+    /// Default to 2 minutes
+    #[structopt(env, default_value = "120")]
+    pub time_to_live: u64,
```

Comment:
Could we either change this to a `Duration` (to remove any ambiguity about the units, which always seem to plague these kind of parameters), or rename it to something like `time_to_live_sec` (or `ttl_sec`)?